media-gfx/hydrus/hydrus-538.ebuild

```diff
@@ -48,19 +50,20 @@ RDEPEND="
 		dev-python/pillow[${PYTHON_USEDEP},lcms]
 		dev-python/psutil[${PYTHON_USEDEP}]
 		dev-python/pyopenssl[${PYTHON_USEDEP}]
-		dev-python/pyside2[widgets,gui,${PYTHON_USEDEP}]
 		dev-python/python-mpv[${PYTHON_USEDEP}]
 		dev-python/pyyaml[${PYTHON_USEDEP}]
 		dev-python/requests[${PYTHON_USEDEP}]
 		dev-python/send2trash[${PYTHON_USEDEP}]
 		dev-python/service-identity[${PYTHON_USEDEP}]
 		dev-python/six[${PYTHON_USEDEP}]
 		dev-python/twisted[${PYTHON_USEDEP}]
+
+		dev-python/QtPy[widgets,gui,svg,multimedia,${PYTHON_USEDEP}]
+		qt5? ( dev-python/QtPy[pyside2] )
+		qt6? ( dev-python/QtPy[pyside6] )
```

Comment:
I wonder if it's just smarter to do `|| ( )` kind of dependency? Just wondering whether portage does the upgrade from pyside2->pyside6 seamlessly with that. 

Would allow you to get rid of `REQUIRED_USE` and `optfeature` conditions. Let me know what you think. Tested this and everything works fine.

Reply:
If that works, it's probably better to do that. Since the program works with both 2 and 6, it should be fine. I'll commit this soon, probably gonna also update the ebuild to the new program version.

Reply:
Seems it won't be as soon as I thought because the new version introduced several dependencies with complex building / testing and several dependencies not in the tree :/

Reply:
Let's just do this change now and I'll figure out the next version later. Pushed.

Reply:
Thanks! I'm still not 100 % sure how portage can deal with the global qt5->qt6 update on this case, but let's adjust if problems arise. 

I'll test today after green CI.